README.md

```diff
@@ -225,32 +225,45 @@ We're going to try spinning this out into its own library; see: https://github.c
 
 2.) Once you have initialized the map (an instance of L.Map), add the following code :
 
-     // Assuming your map instance is in a variable called map and an object with all the map layers is 
+     // Assuming your map instance is in a variable called map and an object with all the map layers is
     in a variable called allMapLayers
     var hash = new L.FullHash(map, allMapLayers);    
-    
-    
-  
+
+
+
  ## Add all LEL Layers at once:
- 
+
  	 L.LayerGroup.EnvironmentalLayers().addTo(map);
-	 
-## Add all layers except some layers: 
+
+## Add all layers except some layers:
 
 	 L.LayerGroup.EnvironmentalLayers({
             exclude: ['mapknitter', 'clouds'],
          }).addTo(map);
-	 
-## Add some layers only: 
+
+## Add some layers only:
 
 	 L.LayerGroup.EnvironmentalLayers({
             include: ['mapknitter', 'clouds'],
          }).addTo(map);
 
-## Turn on Leaflet Hash in the URL: 
+## Turn on Leaflet Hash in the URL:
 
 	 L.LayerGroup.EnvironmentalLayers({
             exclude: ['mapknitter', 'clouds'],
 	    hash: true,             // by default this is FALSE
          }).addTo(map);
-	
+
+## Add an embed control for embedding the map in other pages :
+
+Add the following code after you have the map(an instance of L.Map) initialized:
+
+    // Assuming your map instance is in a variable called map
+    var embedControl = new L.control.embed(options);
+    embedControl.addTo(map);
+
+The optional options object can be passed in with the any of the following properties:
```

Comment:
There seems to be a small typo. Should be good to go once you remove the extra 'the' that comes before the word 'any' on line 265. Thanks!